RiotSharp/RiotSharp.nuspec

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>$id$</id>
+    <version>$version$</version>
+    <title>$title$</title>
+    <authors>$author$</authors>
+    <owners>$author$</owners>
+    <licenseUrl>https://github.com/BenFradet/RiotSharp</licenseUrl>
+    <projectUrl>https://github.com/BenFradet/RiotSharp/blob/develop/LICENSE.md</projectUrl>
+    <iconUrl>https://raw.githubusercontent.com/BenFradet/RiotSharp/develop/logo.png</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>$description$</description>
+    <releaseNotes>Migrated to V3.</releaseNotes>
```

Comment:
mmmh I'm keen on not hardcoding this and letting it be specified when running `nuget pack`, what do you think?